tests/Microsoft.Graph.DotnetCore.Core.Test/Requests/Content/BatchRequestContentTests.cs

```diff
@@ -261,7 +273,7 @@ public async System.Threading.Tasks.Task BatchRequestContent_GetBatchRequestCont
                                   "          \"timeZone\": \"Pacific Standard Time\"\r\n" +
                                   "        },\r\n" +
                                   "        \"location\": {\r\n" +
-                                  "          \"displayName\": \"Harry's Bar\"\r\n" +
+                                  "          \"displayName\": \"Harrys Bar\"\r\n" +
```

Comment:
Is there no escape character for single ticks?

Reply:
More info is here https://docs.microsoft.com/en-us/dotnet/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to#minimal-character-escaping.

The single tick character will be escaped to emit `\u0027`